source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,176 @@
+- Select the **Channel options** icon in the sidebar, then select **Create New Category**.
+
+Next, type a category name, select **Create**, then drag any channels or Direct Messages into this new category. You can also multi-select channels and Direct Messages to drag them together as a group.
+
+Making categories work for you
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Categories are collapsible**
+
+- When collapsed, only unread channels display to reduce unnecessary scrolling.
+- When expanded, all channels in the category display, including channels with unread messages.
+
+**You can reorder categories**
+
+- Drag to reorder entire categories to prioritize important conversations. 
+
+**Categories can contain Direct Message conversations**
+
+- Select and drag Direct Messages into any category. You can also multi-select Direct Messages to drag them together as a group.
+
+    .. image:: ../../images/sidebar_ga.gif
+
+Renaming categories
+^^^^^^^^^^^^^^^^^^^
+
+1. Select the **Category options** icon in the sidebar, then select **Rename Category**.
+2. Type a new category name, then select **Rename**.
+
+Deleting categories
+^^^^^^^^^^^^^^^^^^^
+
+1. Select the **Category options** icon in the sidebar, then select **Delete Category**.
+2. Select **Delete** to confirm or select **X** to cancel.
+
+All channels and Direct Message conversations in the deleted category move back to their default **Channels** and **Direct Messages** categories. Deleting a category never removes you from channels you have joined. 
+
+Organizing channels in categories
+---------------------------------
+
+Once you've created categories, you can move channels within them to organize your sidebar.
+
+Multi-selecting channels and Direct Messages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using the Mattermost Web or Desktop App:
+
+- Select sequential channels and/or Direct Messages by pressing and holding SHIFT. 
+- Select non-sequential channels and/or Direct Messages by pressing and holding CMD (for Mac) or CTRL (for Windows/Linux). 
+- Press ESC to clear channel or Direct Message selections.
+
+Multi-selected channels and Direct Messages move together as a group in the order they originally appeared.
+
+Dragging and dropping selections
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using the Mattermost Web or Desktop App, drag selected channels and/or Direct Messages between or within categories. 
+
+    .. image:: ../../images/multi-select-drag.gif
+
+Moving selections
+^^^^^^^^^^^^^^^^^
+
+In addition to selecting and dragging, you can also specify a destination for selected channels and/or Direct Messages using the **Move to** option under the **Channel options** icon in the sidebar.  
+
+    .. image:: ../../images/multi-select-move.gif
+
+Muting and unmuting categories
+------------------------------
+
+When you mute or unmute a category, all channels within that category are also muted or unmuted. 
```

Comment:
When you mute or unmute a category, all channels within that category are also muted or unmuted. You can still selectively unmute specific channels within a muted category.